vro-types/o11n-plugin-ssh/index.d.ts

```diff
@@ -175,81 +175,117 @@ declare class SSHSession {
 	 * @param basePath
 	 * @param pattern
 	 */
-	findAll(basePath: any, pattern: string): string[];
+	findAll(basePath: string, pattern: string): string[];
 	/**
 	 * Recursively search and return matching pattern file ONLY. This command will connect the session, request an execution channel, execute the command and close the execution channel. The session is open automatically if not opened yet, but will remain open after the command execution. Use disconnect to close the session.
 	 * @param basePath
 	 * @param pattern
 	 */
-	findFile(basePath: any, pattern: string): string[];
+	findFile(basePath: string, pattern: string): string[];
 	/**
 	 * Recursively search and return matching pattern directory ONLY.  This command will connect the session, request an execution channel, execute the command and close the execution channel. The session is open automatically if not opened yet, but will remain open after the command execution. Use disconnect to close the session.
 	 * @param basePath
 	 * @param pattern
 	 */
-	findDir(basePath: any, pattern: string): string[];
+	findDir(basePath: string, pattern: string): string[];
 	/**
 	 * List files and directories in path. This command will connect the session, request an execution channel, execute the command and close the execution channel. The session is open automatically if not opened yet, but will remain open after the command execution. Use disconnect to close the session.
 	 * @param basePath
 	 */
-	listAll(basePath: any): string[];
+	listAll(basePath: string): string[];
 	/**
 	 * List files ONLY in path. This command will connect the session, request an execution channel, execute the command and close the execution channel. The session is open automatically if not opened yet, but will remain open after the command execution. Use disconnect to close the session.
 	 * @param basePath
 	 */
-	listFile(basePath: any): string[];
+	listFile(basePath: string): string[];
 	/**
 	 * List directories ONLY in path. This command will connect the session, request an execution channel, execute the command and close the execution channel. The session is open automatically if not opened yet, but will remain open after the command execution. Use disconnect to close the session.
 	 * @param basePath
 	 */
-	listDir(basePath: any): string[];
+	listDir(basePath: string): string[];
 	/**
 	 * Disconnects the current session if open. The session parameters are given in the object constructor (host, username, password); a session is opened automatically when any method is called that requires a session (like execute). Once open, the session remains open; any method uses the same session but opens a new channel and the closes it when done. You need to use this disconnect method to close the session.
 	 */
-	disconnect(): void;
+	disconnect(): object;
```

Comment:
Last one :) I somehow missed it..
```suggestion
	disconnect(): any;
```